amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -4018,6 +4223,126 @@ definitions:
     properties:
       reason:
         type: string
+  RetentionNodeBodyCreate:
+    type: object
+    properties:
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
+        type: boolean
+  RetentionScheduleResponse:
+    type: object
+    properties:
+      id:
+        type: string
+      parentId:
+        type: string
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
+        type: boolean
+      unpublishedUpdates:
+        type: boolean
+  RetentionScheduleResponseList:
+    type: object
+    properties:
+      entries:
+        type: array
+        items:
+          $ref: '#/definitions/FullRetentionScheduleResponse'
+  FullRetentionScheduleResponse:
+    type: object
+    properties:
+      id:
+        type: string
+      parentId:
+        type: string
+      authority:
+        type: string
+      instructions:
+        type: string
+      isRecordLevel:
+        type: boolean
+      unpublishedUpdates:
+        type: boolean
+      actions:
+        type: array
+        items:
+          $ref: '#/definitions/Actions'
+      events:
+        type: array
+        items:
+          type: string
+  Actions:
+    type: object
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      periodAmount:
+        type: integer
+      period:
+        type: string
+      periodProperty:
+        type: string
+      combineDispositionStepConditions:
+        type: boolean
+      eligibleOnFirstCompleteEvent:
+        type: boolean
+      description:
+        type: string
+      destroyWithGhosting:
+        type: boolean
+      location:
+        type: string
+      index:
+        type: integer
+  RetentionStepNodeBodyCreate:
+    type: object
+    properties:
+      name:
+        type: string
+      periodAmount:
+        type: integer
+      period:
+        type: string
+      periodProperty:
+        type: string
+      combineDispositionStepConditions:
```

Comment:
Since this is usually not valid then can we have a default value, or mark it as optional?

Reply:
As per the comment we have made this property default value as false.

Reply:
I thought we were also going to have a description of this property here.

Reply:
Yes we have added the description in the swagger.